src/neptune_fastai/impl/__init__.py

```diff
@@ -154,7 +154,9 @@ def _batch_size(self) -> int:
 
     @property
     def _optimizer_name(self) -> Optional[str]:
-        return self.opt_func.__name__
+        optim_name = getattr(self.opt_func, "__name__", "NA")
+        warnings.warn("NeptuneCallback: Couldn't retrieve the optimizer name, so it will logged as 'NA'.")
```

Comment:
"NeptuneCallback: Couldn't retrieve the optimizer name, so it will be logged as 'NA'."

or should it be 'N/A'? not sure which option will be consistent with other instances of this in Neptune, but usually this abbreviation includes a slash :)

Reply:
Can we not skip logging this field instead of logging it as NA? 

The warning can also be updated to 

"NeptuneCallback: Couldn't retrieve the optimizer name, so it will not be logged.
You can set the optimizer name by passing it to the `__name__` attribute:
`>>> optimizer.__name__ = "NAME"`
"

Reply:
The reason, I thought of logging something like `NA` was that if later the user has a script which fetches values from run then this would fail (So changing the structure would be BC-breaking). Wdyt?

Reply:
Makes sense (y)

Reply:
Will update the message to 

`"NeptuneCallback: Couldn't retrieve the optimizer name, so it will be logged as "N/A". You can set the optimizer name by assigning it to the __name__ attribute. Eg. >>> optimizer.__name__ = 'NAME'`

Thanks!